pkg/agent/openflow/groups.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openflow
+
+import (
+	"sync"
+
+	binding "antrea.io/antrea/pkg/ovs/openflow"
+)
+
+type GroupAllocator interface {
+	New() binding.GroupIDType
+	Recycle(id binding.GroupIDType)
```

Comment:
"New" is widely used to instantiate a struct, rarely for resource allocation. Is it more proper to use `Allocate` and `Release`?

Reply:
`New` and `Recycle` are actually the existing API names used for AntreProxy, and I just move the function to a new path but don't change the logics. 

You are correct, `Allocate` and `Release` are more proper for id allocater use cases. I would rename them and update.